molgenis_documentation/user_documentation.md

```diff
@@ -947,6 +951,163 @@ To get the feel of how certain styles look, you can select it in the dropdown me
+
+###### <a name="upload-screens-options-steps"></a>Steps
+
+1. Click on the next Button.
+
+![Upload file screen](images/plugin-upload/options-screen.png?raw=true, "Options")
+
+##### <a name="upload-screens-packages"></a>Packages page
+Because the entity (table) persons has no package defined, the by the packages page we have the posibility to choose another package then the MOLGENIS default package.
+The select options are generated from the available existing packages in this data set.
+
+###### <a name="upload-screens-packages-steps"></a>Steps
+
+1. Click on the next Button.
+
+![Upload file screen](images/plugin-upload/packages-screen.png?raw=true, "Packages")
+
+##### <a name="upload-screens-validation"></a>Validation page
+When you see this page the validation is already done. This page validates the structure of the meta data.
+
+"Enitities" table where all the entities (tables) are defined.
+
+* Name: Name of entity
+* Importable: Is this entity inportable or not. Two options (Yes, No) 
+
+"Entity fields" table that will contain information a bout the fields of a entity (Columns of the table)
+
+* Name: Name of entity
+* Detected: A comma seperated list of fields that where found for this entity.
+* Requierd: Are there requierd fields defined in the meta data that are missing in the entity?
+* Available: Are there fields in the meta data that are optionel and where not found in the antity?
+* Unknown: Are there fields defined in the entity the where undefined in the meta data?
+
+###### <a name="upload-screens-validation-steps"></a>Steps
+
+1. Click on the next Button.
+
+![Upload file screen](images/plugin-upload/validation-screen.png?raw=true, "Validation")
+
+##### <a name="upload-screens-result"></a>Result page
+
+When this page is shown with the "import succes" message, than you know that your data and metadata are uploaded correctly.
+
+After the data is uploaded into MOLGENIS, you can change some permission for the entities.
+
+In the permission view you can:
+
+1. Select a group: which user group will get this permissions.
+2. Select permission for a entity (table). You can choose between: Edit, View, Count and None. For more information about permissions visit this section: [Setting permissions](#permissions)
+
+###### <a name="upload-screens-result-steps"></a>Steps
+
+1. [Optional] Select the view permission for the root_hospital_cities entity.
+2. [Optional] Click on the save button.
+3. Click on the finish button.
+
+![Upload file screen](images/plugin-upload/result-screen.png?raw=true, "Result")
+
+##### <a name="dataexplorer-reports"></a>Data-explorer "Reports"
+The reports functionality is made for overriding the default row or rows view in the Data-explorer. Overriding the views is only possible in development environment before the application is deployed into the server. In this short tutorial I will show you how to achieve this.
+
```

Comment:
I do not understand this part. The report functionality gives you an extra tab, it is not replacing the default rows view. And it can also be used fine on production servers, reports can even be added runtime to the database.